utils/cppcheck.sh

```diff
@@ -26,26 +26,20 @@ cppcheck "$@" \
   -I build/lib \
   -I enterprise \
   -I lib \
-  -D USE_PLAN_CACHE \
   -D DEFINE_FACTORY_DEFAULT \
   --std=c++20 \
   --enable=warning,performance,portability,missingInclude \
   --force \
   --quiet \
   --platform=unix64 \
   --inline-suppr \
-  --suppress="*:*yacc.c*" \
```

Comment:
Please suppress `tokens.ll` and `grammar.cpp` they are auto generated

Reply:
They are already suppressed.